com/win32comext/mapi/src/PyIMsgStore.i

```diff
@@ -79,6 +79,32 @@ PyObject *PyIMsgStore::OpenEntry(PyObject *self, PyObject *args)
 
 %}
 
+%native(StoreLogoff) StoreLogoff;
+%{
+// @pyswig <o PyIInterface>|StoreLogoff|Enables the orderly logoff of the message store.
+PyObject *PyIMsgStore::StoreLogoff(PyObject *self, PyObject *args) 
+{
+    HRESULT  _result;
```

Comment:
It looks like there are some mixed tabs and spaces here - do you mind fixing that?

Reply:
FWIW, tools like proposed in this PR #2034 could autofix these kinds of issues